source/_data/imageStyles.yaml

```diff
@@ -12,3 +12,6 @@ imageStyles:
     width: 800
   large:
     width: 1600
+  carousel:
```

Comment:
This seems like a strange image size/ratio. Can we better rely on a more standard width and then grab a corresponding height based on a 3:2 ratio or other standard?